templates/sway-script-test-rs/README.md

```diff
@@ -0,0 +1,11 @@
+# Rust + Sway integration testing
+
+A cargo-generate template that makes it easy to initialise Rust integration
+testing within an existing Sway project.
+
+This template is designed for Rust developers who wish to test integration of
+their Rust application and Sway code.
+
+See the [Testing with Rust](https://fuellabs.github.io/sway/master/book/testing/testing-with-rust.html) chapter of [the Sway
```

Comment:
should probably link to the docs hub here also

Reply:
The docs hub url is docs.fuel.network

```suggestion
See the [Testing with Rust](https://docs.fuel.network/docs/sway/testing/testing-with-rust/) chapter of [the Sway
```